jni/src/faiss_wrapper.cpp

```diff
@@ -461,6 +462,78 @@ jlong knn_jni::faiss_wrapper::LoadIndexWithStream(faiss::IOReader* ioReader) {
 
     return (jlong) indexReader;
 }
+jlong knn_jni::faiss_wrapper::LoadIndexWithStreamADCParams(faiss::IOReader* ioReader, knn_jni::JNIUtilInterface * jniUtil, JNIEnv * env, jobject methodParamsJ) {
+    auto methodParams = jniUtil->ConvertJavaMapToCppMap(env, methodParamsJ);
+
+    auto quantization_level_it = methodParams.find("quantization_level");
+    knn_jni::QuantizationLevel quantLevel = knn_jni::QuantizationLevel::NONE;
+    if (quantization_level_it != methodParams.end()) {
+        quantLevel = jniUtil->ConvertJavaStringToQuantizationLevel(env, quantization_level_it->second);
+    } else {
```

Comment:
nit: In general, to simplify code, try to avoid else's if you can. For example:
```
if (quantization_level_it != methodParams.end()) {
    throw std::runtime_error("Quantization level not specified in params");
}
```

The fewer {}, the better. @VijayanB  is a big proponent of this.